sunpy/physics/limb_darkening.py

```diff
@@ -1,32 +1,30 @@
 import numpy as np
 from astropy import units as u
 from astropy.coordinates import SkyCoord
+import copy
 
 
-def darklimb_u(wavelength):
+@u.quantity_input
+def __darklimb_u(wavelength):
```

Comment:
We should only be using a single underscore to indicate "private", also you need to annotate the argument with the expected unit:
```suggestion
def _darklimb_u(wavelength: u.AA):
```